src/fd_table.c

```diff
@@ -307,7 +313,7 @@ uvwasi_errno_t uvwasi_fd_table_insert_preopen_socket(uvwasi_t* uvwasi,
                                 NULL,
                                 NULL,
                                 UVWASI_FILETYPE_SOCKET_STREAM,
-                                UVWASI__RIGHTS_SOCKET_BASE,
+                                UVWASI__RIGHTS_SOCKET_BASE | UVWASI_RIGHT_SOCK_ACCEPT,
```

Comment:
Should `UVWASI_RIGHT_SOCK_ACCEPT` be part of `UVWASI__RIGHTS_SOCKET_BASE`? I'm not sure how much it matters because I believe the whole rights system has gone away in future versions of WASI.

Reply:
Added UVWASI_RIGHT_SOCK_ACCEPT be part of UVWASI__RIGHTS_SOCKET_BASE